backend/src/managers/TranslationManager.ts

```diff
@@ -156,6 +167,12 @@ export default class TranslationManager {
             return Promise.resolve();
         }
 
+        const readableGPTStream = await TranslationManager.interpreterString(prompt, content, temp);
+
+        // Important!
+        // Ensure that the first chunk is written after the stream is created,
+        // otherwise the error cannot be set in http response downstream
```

Comment:
It can - see comment above.

Reply:
Preserving this invariant when we can works better.